docs/interactions/Receiving_and_Responding.md

```diff
@@ -25,11 +25,15 @@ For [Message Components](#DOCS_INTERACTIONS_MESSAGE_COMPONENTS/) it includes ide
 | token          | string                                                                                                        | a continuation token for responding to the interaction         |
 | version        | integer                                                                                                       | read-only property, always `1`                                 |
 | message?       | [message](#DOCS_RESOURCES_CHANNEL/message-object) object                                                      | for components, the message they were attached to              |
+| locale?\*\*\*  | string                                                                                                        | the selected language of the invoking user                     |
```

Comment:
why isn't it on the user object?

Reply:
when a user uses an interaction they are providing data to the application; being able to access the data arbitrarily is more of a privacy issue

Reply:
I think adding it to the user object would be a bad design. This is not a canonical user property and having a "interaction user" or something (a la thread member) would be complecting stuff.

Reply:
![image](https://user-images.githubusercontent.com/14029133/146821724-9c95af71-5eba-474e-bc5c-7427e5c96373.png)
I have to disagree

Reply:
That field is only sent for the current user (ie `GET /users/@me`) which most libraries already treat as a separate structure.

Reply:
What are the edge cases where `locale` is not provided? As far as I can see, it is always provided. The same for `guild_locale` it seems to be always en-US for non-community guilds.

Reply:
Guild locale is missing iirc in DMs instead of being null.
But I'm not sure.

Reply:
@Victorsitou that is answered in a note below the table

Reply:
@advaith1 oh, I completely missed that, thanks!

Reply:
So `locale` is expected to always be present (except for `PING` interactions) and `guild_locale` for all interactions on a guild, right?